packages/apollo-dx/package.json

```diff
@@ -10,9 +10,7 @@
         "dist": "yarn run rollup && yarn run uglify",
         "build": "yarn run lint && yarn run babel && yarn run prepare-build && yarn run dist",
         "publish-script": "node ../../scripts/publish.js",
-        "test": "jest",
-        "lint": "eslint --ext js,jsx ./src",
```

Comment:
How does "yarn build" work if you remove "lint" script here ?

Reply:
:o shocked. How does it work right now ?